types/ramda/index.d.ts

```diff
@@ -1181,7 +1184,7 @@ declare namespace R {
          * Returns a partial copy of an object containing only the keys specified.  If the key does not exist, the
          * property is ignored.
          */
-        pick<T, U>(names: string[], obj: T): U;
+        pick<T, K extends keyof T>(names: Array<K | string>, obj: T): Pick<T, K>;
```

Comment:
The string alternative must be there because it is possible to call the pick function with names which are not part of the supplied object.
There are a couple of tests that show this.